docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -36,38 +39,12 @@ ocp_local_config_path=/tmp/ocp
 connection=local
 ssh_cmd=""
 ----
++
+<1> Instructs os-diff to access your {OpenStackPreviousInstaller} host through SSH. The default value is `ssh -F ssh.config`. However, you can set the value without an ssh.config file, for example, `ssh -i /home/user/.ssh/id_rsa stack@my.undercloud.local`.
```

Comment:
@matbu Can you please review the latest iteration of step 1? Looking at the preview might help:
https://logserver.rdoproject.org/86/586/a4d03195480a426bd0820f20d458a8e76818cbee/github-check/adoption-docs-preview/b423910/docs_build/docs_build/adoption-user/index-downstream.html#comparing-configuration-files-between-deployments_storage-requirements

Reply:
LGTM